README.md

```diff
@@ -149,7 +147,7 @@ No PECL extensions is required. The [Go! AOP](http://go.aopphp.com/) library doe
 
 ## Requirements
 
-PHP >= 5.6 + [Go! AOP Requirements](https://github.com/goaop/framework#requirements)
+PHP >= 7.4 [Go! AOP Requirements](https://github.com/goaop/framework#requirements)
```

Comment:
There is no Requirements section in goaop/framework README, so please remove the link.
And it doesn't support PHP 8 yet.